src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.scss

```diff
@@ -200,10 +200,68 @@
       line-height: 16px;
       color: #6F6F6F;
     }
+
+     .publish-date {
+      font-family: 'Encode Sans', sans-serif;
+      font-size: 13px;
+      font-weight: 400;
+      line-height: 15.56px;
+      letter-spacing: 0.08em;
+      text-align: left;
+      color: #B7B7B7;
+    }
+
+    .partner-responses {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-bottom: 20px !important;
+      margin-top: 20px;
+
+
+      .partner-logo {
+        width: 40px;
+        height: 31px;
+        object-fit: contain;
+        border-radius: 5px;
+      }
+
+      .partner-details {
+        margin-left: 10px;
+      }
+
+      .partner-name {
+        font-family: 'Encode Sans', sans-serif;
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 16.76px;
+        letter-spacing: 0.035em;
+        text-align: left;
+        color: #151515;
+        text-decoration: underline;
+      }
+
+      .response-count {
+        font-size: 13px;
+        color: #666;
+        font-weight: 700;
+        line-height: 16.25px;
+        font-family: 'Encode Sans', sans-serif;
```

Comment:
Add the font-family in variables and use the variable everywhere bro

eg: `$encode-sans: 'Encode Sans', sans-serif;`